schema/algolia-reverse-timestamp-schema.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
maybe it is worth to put attributesForFaceting and attributesToIndex into one place?